### PR TITLE
cargo-build: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/common/cargo-build.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+- Require that `Cargo.lock` is up to date:
+
+`cargo build --locked`
+
+- Build all packages in the workspace:
+
+`cargo build --workspace`
+
+- Build a package:
```
Comment (sbrl) [inline pages/common/cargo-build.md:18]: Perhaps this will read better?
```suggestion
- Build a specific package:
```

**Comment by Alfriadox** [comment]
Would it be possible to have the basic `cargo build` invocation (no args) first here?

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel :smiley_cat:

**Comment by navarroaxel** [comment]
Could @schneiderl or @einverne please review?

**Review by schneiderl — APPROVED** [review]

**Pull request merged**